lib/explorer/polars_backend/series.ex

```diff
@@ -68,31 +51,14 @@ defmodule Explorer.PolarsBackend.Series do
   # Introspection
 
   @impl true
-  def dtype(series), do: series |> Shared.apply_series(:s_dtype) |> Shared.normalise_dtype()
+  def dtype(series), do: series |> Shared.apply_series(:s_dtype)
 
   @impl true
   def size(series), do: Shared.apply_series(series, :s_size)
 
   @impl true
   def iotype(series) do
-    case Shared.apply_series(series, :s_dtype) do
-      "u8" -> {:u, 8}
-      "u32" -> {:u, 32}
-      "i32" -> {:s, 32}
-      "i64" -> {:s, 64}
-      "f64" -> {:f, 64}
-      "bool" -> {:u, 8}
-      "date" -> {:s, 32}
-      "time" -> {:s, 64}
-      "datetime[ms]" -> {:s, 64}
-      "datetime[μs]" -> {:s, 64}
-      "datetime[ns]" -> {:s, 64}
-      "duration[ms]" -> {:s, 64}
-      "duration[μs]" -> {:s, 64}
-      "duration[ns]" -> {:s, 64}
-      "cat" -> {:u, 32}
-      dtype -> raise "cannot convert dtype #{inspect(dtype)} to iotype"
-    end
+    Shared.apply_series(series, :s_iotype)
```

Comment:
No changes required now but, for completeness, we want to support u8, u32, etc as dtypes themselves, which means we will be able to move this function all the way up to `Explorer.Series` and remove the callback altogether in the future. :)